org.eclipse.jdt.ls.target/org.eclipse.jdt.ls.tp.target

```diff
@@ -43,7 +43,7 @@
             <repository location="http://download.jboss.org/jbosstools/updates/m2e-extensions/m2e-apt/1.5.2-2018-12-24_15-46-05-H18/"/>
         </location>
 	    <location includeAllPlatforms="false" includeConfigurePhase="false" includeMode="planner" includeSource="true" type="InstallableUnit">
-		    <repository location="http://services.typefox.io/open-source/jenkins/job/lsp4j/job/master/lastSuccessfulBuild/artifact/build/p2-repository/"/>
+		    <repository location="https://github.com/akaroml/lsp4j-latest/raw/master/p2/"/>
```

Comment:
The PR to LSP4J is in review eclipse/lsp4j#362. This link is to reference the private build for testing purpose.
I'll revert this line once the LSP4J PR is merged.

Reply:
eclipse/lsp4j#362 is merged. Reverted back to the typefox endpoint.